ui/src/components/inspectExamples/InspectExamples.js

```diff
@@ -5,7 +5,7 @@ import {TwitterShareButton, TwitterIcon} from 'react-share';
 
 import './inspectExamples.css';
 import httpRequests from '../../helpers/httpRequests';
-import config from '../../helpers/config';
+//import config from '../../helpers/config';
```

Comment:
If something is not needed anymore, please remove, or add a further comment why this might be needed in the future.